configure

```diff
@@ -732,15 +734,44 @@ else
 	fi;
 fi
 
-if [ -z "$systemdsystemunitdir" ]; then
-	if $PKG_CONFIG --exists systemd; then
-		systemdsystemunitdir=$($PKG_CONFIG --variable=systemdsystemunitdir systemd)
+if ! $PKG_CONFIG --exists systemd; then
+    if [ -n "$systemdsystemunitdir" ] ; then
```

Comment:
it seems odd that requesting systemd support is done by specifying a directory

Reply:
The idea is if you request systemd support explicitly there will be an error if it is not found.
Specifying the `$systemdsystemunitdir` was the only way to do that.